### DISCUSSION TITLE
Carto mobile sdk vs MapLibre

### DISCUSSION THREAD

**Discussion by safe-bug** [maplibre/maplibre-native#2690] [Q&A]
Hello,

I currently have an application built on the Carto mobile SDK, but since Carto has stopped supporting this project, I'm exploring alternative options. I'm particularly interested in hearing from anyone with experience switching to MapLibre.

Could you share the key benefits and drawbacks of using MapLibre compared to Carto? Any insights or advice would be greatly appreciated.

**Comment by louwers** [comment]
I am not familiar with Carto products. What is your use case exactly?

It seems like their Mobile SDK has built-in geocoding and navigation. These are not part of our base SDK. MapLibre Native only supports Web Mercator. We do have pretty good offline support that is widely used. For styling you would need to use the MapLibre Style Spec: https://maplibre.org/maplibre-style-spec/ You can also see what data sources are supported there, but that looks pretty similar to Carto. There are many services available that can be used with MapLibre, hosting your own tiles is also possible.

**Reply by safe-bug** [reply]
Thanks for your reply
Our product is a maps and navigation app, and we're hosting our own set of services. So, as you mentioned, there shouldn't be many downsides. As an example I thought we might need to develop our own tile caching mechanisms and similar features.

**Comment by sjg-wdw** [comment]
Oh man, I've been in this industry too long.  I remember when CartoDB bought Nutiteq and they turned their toolkit into the Carto mobile SDK.  They were competitors of mine and they gave off some very weird vibes at conferences.  Nutiteq, not Carto.  They were always cool. 

I met with Javier in SF just a few weeks ago and he mentioned this in passing.  It's fun to see it here.

MapLibre Native is what you want.  It is extremely serious, trusted by giant companies like Amazon and Meta and in use by so very many consumers who aren't even aware of it.  It's different than what you're used to, but you won't regret switching.

**Reply by safe-bug** [reply]
Yeah, exactly. We we're also using Nutiteq and then had to switch to Carto. Thanks very much
